velox/type/Timestamp.h

```diff
@@ -191,6 +192,31 @@ struct Timestamp {
   std::chrono::time_point<std::chrono::system_clock, std::chrono::milliseconds>
   toTimePoint(bool allowOverflow = false) const;
 
+  static Timestamp fromSeconds(double seconds) {
```

Comment:
Would you check Spark's implementation and handle NaN and Inf accordingly?
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala#L596-L601